src/pages/partners.tsx

```diff
@@ -140,8 +144,15 @@ export default function partners(): ReactElement {
 						<TabPanels mt={10}>
 							<TabPanel>
 								<SimpleGrid
-									minChildWidth="250px"
-									spacing="40px"
+									columns={[2, null, 3]}
```

Comment:
```suggestion
									columns={{base: 2, md: null, lg: 3}}
```

Don't use the array syntax; instead, use the object syntax
(Unsure if this is the equivalent, please verify)